ark/phenotyping/pixel_consensus_cluster.R

```diff
@@ -38,23 +40,27 @@ clusterAvgPath <- args[6]
 # get consensus clustered write path
 pixelMatConsensus <- args[7]
 
+# get the clust to meta write path
+clustToMeta <- args[8]
+
 # set the random seed
-seed <- strtoi(args[8])
+seed <- strtoi(args[9])
 set.seed(seed)
 
 # read cluster averaged data
 print("Reading cluster averaged data")
-clusterAvgs <- arrow::read_feather(clusterAvgPath)
+clusterAvgs <- as.data.frame(read.csv(clusterAvgPath, check.names=FALSE))
 
 # scale and cap the data accordingly
-print("Scaling data")
-clusterAvgsScale <- pmin(scale(clusterAvgs[markers]), cap)
+# NOTE: z-scoring and capping cluster avg data produces better clustering results
+clusterAvgsScale <- pmin(scale(clusterAvgs[,markers]), cap)
```

Comment:
I think you are z-scoring the data twice. The scale function here z-scores the data, but I think you are already z-scoring the data in compute_pixel_cluster_channel_avg, which is written to pc_chan_avg_som_cluster_name.

Reply:
Ah good point, this changed when we ported this over to Python.